src/geofabrics/__init__.py

```diff
@@ -4,4 +4,4 @@
 
 @author: pearsonra
 """
-__version__ = "0.1.0"
```

Comment:
I wonder if there is a way to automate or lint for this. Maybe a suggestion to look into as the project expands

Reply:
Cool. Good suggestion. I will leave it for now though - but will reconsider when getting close to the first release: "1.0.0". I had a quick internet search and relevant examples appear to be: 

- https://stackoverflow.com/questions/6786555/automatic-version-number-both-in-setup-py-setuptools-and-source-code
- https://stackoverflow.com/questions/5581722/how-can-i-rewrite-python-version-with-git
- Blog discussing a couple of options: https://hackernoon.com/lets-automate-version-number-updates-not-a91q3x7n